### PR TITLE
Nwd nnw

### PR DESCRIPTION
**Pull request by lubankp**
@lubankp added homework nwd-nnw

### CODE REVIEW

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @lubankp

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @lubankp

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,37 @@
...
+    if (lhs == 0){
+        range = std::abs(rhs);
+    }else if(rhs == 0){
+        range = std::abs(lhs);
+    }else if(lhs > rhs){
+        range = std::abs(rhs);
+    }else{
+        range = std::abs(lhs);
+    }
+
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:18]: Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,37 @@
...
+    for (auto i=1; i <= range; ++i){
+        if(lhs%i == 0 and rhs%i == 0){
+            result = i;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:21]: Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..

File: homework/nwd-nnw/nwdNww.hpp [hunk 3]
```diff
@@ -1,11 +1,37 @@
...
 
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    
+    auto result = NWD(lhs,rhs);
+    auto multiple = 0;
+    if (result != 0){
+        multiple = std::abs(lhs * rhs) / result;
+    }
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:35]: Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**